portfolio/.eslintrc.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
This is from the linter changes.  If you already merged those in, just rebase this branch